freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/KerberosMgmt/doc/KeytabModelNotes.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.freeipa.api.v1.KerberosMgmt.doc;
+
+public class KeytabModelNotes {
+    public static final String CREATE_HOST_AND_PRINCIPAL_NOTES  = "Creates the host if it doesn't exist and also creates the service principal for the given " +
+            "service";
+    public static final String GET_KEYTAB_NOTES = "Retrieves the keytab for the service principal derived from the host and service provided. " +
```

Comment:
You may want to mention that it "Retrieves the existing keytab .... ".  It is important that the keytab already exists otherwise this API will fail.

Reply:
done.